packages/inscription/src/inscription/key-listener.ts

```diff
@@ -0,0 +1,37 @@
+import { FocusDomAction, GlobalKeyListenerTool, matchesKeystroke, SetAccessibleKeyShortcutAction } from '@eclipse-glsp/client';
+import { ToggleInscriptionAction } from './action';
+import { injectable } from 'inversify';
+
+@injectable()
+export class IvyInscriptionGlobalKeyListenerTool extends GlobalKeyListenerTool {
+  registerShortcutKey(): void {
+    this.actionDispatcher.onceModelInitialized().then(() => {
+      this.actionDispatcher.dispatchAll([
+        SetAccessibleKeyShortcutAction.create({
+          token: 'Inscription',
+          keys: [{ shortcuts: ['CTRL', '3'], description: 'Focus on inscription', group: 'Inscription', position: 0 }]
```

Comment:
why like this? why not 'mod+3'?

and already saw this in the pr description. isn't it possible to do this with 1-3, because it is like that on the other editors...

Reply:
ah this is not handled via the useShortcut right?

Reply:
it is a global tool that always listens, even if an input is focused...